src/index.ts

```diff
@@ -166,6 +226,50 @@ async function main() {
     process.exit(1);
   }
 
+  try {
+    if (conf.useMercury && redis) {
+      const checkNetwork = "PUBLIC";
+      // initial token and userID not needed for integrity checks
+      const integrityCheckMercurySession = {
+        renewClientMaker,
+        backendClientMaker,
+        currentDataClientMaker,
+        backends,
+        credentials: {
+          TESTNET: {
+            email: conf.mercuryEmailTestnet,
+            password: conf.mercuryPasswordTestnet,
+          },
+          // need to set this to the integrity check accounts for Mercury to remove entries periodically
+          PUBLIC: {
+            email: conf.mercuryIntegrityCheckEmail,
+            password: conf.mercuryIntegrityCheckPass,
+          },
+        },
+      };
+      const integrityCheckMercuryClient = new MercuryClient(
+        integrityCheckMercurySession,
+        logger,
+        register,
+        {
+          mercuryErrorCounter,
+          rpcErrorCounter,
+          criticalError,
+        },
+        redis
+      );
+      const stellarClient = new IntegrityChecker(
```

Comment:
NIT: should this have a more informative name like `integrityCheckerClient` instead of `stellarClient`?

Reply:
oh yup thanks! Renamed in b097353d916e1e15d18ce904e6846afb230fa1fd